src/Tempest/Http/src/RouteConfig.php

```diff
@@ -40,25 +52,24 @@ public function addRoute(MethodReflector $handler, Route $route): self
         return $this;
     }
 
-    /**
-     * Build one big regex for matching request URIs.
-     * See https://github.com/tempestphp/tempest-framework/pull/175 for the details
-     */
-    private function addToMatchingRegex(Route $route, string $routeMark): void
+    public function prepareMatchingRegexes(): void
```

Comment:
During the Route discovery we do not yet create the matchingRegexes, doing so would be wastefull. That is why we expose a separate method to create these regexes right before we need them. Currently this is done:

- before route matching
- before serialization (in `__sleep`)